launch/LMS1xx.launch

```diff
@@ -1,6 +1,13 @@
 <launch>
   <arg name="host" default="192.168.1.14" />
+  
+  <!-- start and end units are in degrees to keep with lms internal structure. They will be converted to rad in the node. -->
+  <arg name="start" default="-999.9" />
+  <arg name="end" default="999.9" />
```

Comment:
If the inputted values are outside of the range of the sensor then it uses the full range. I put +/-999 as some arbitrary large numbers outside of the bounds.